Zoo/Model/Animals/Amphibia/Alligator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Zoo.Util.Enum;
+
+namespace Zoo.Model.Animals.Amphibia
+{
+    internal sealed class Alligator : AmphiniaAnimal
+    {
+        private readonly string Sound = "MmmmmmHeemmmmm";
```

Comment:
It more correct to set the DM value in the constructor.
```suggestion
        private readonly string Sound;
        
        public Alligator()
        {
            ...
            this.Sound = "MmmmmmHeemmmmm";
        }
```